html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-history-pushState.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<title>History state change for iframe loading='lazy' before it is loaded: history.pushState</title>
+<iframe data-src="about:blank#foo" src="support/blank.htm?src" loading="lazy"></iframe>
```

Comment:
```suggestion
<iframe data-src="about:blank#push" src="support/blank.htm?src" loading="lazy"></iframe>
```

nit: more descriptive